docs/migrate/migrate-guide.rst

```diff
@@ -0,0 +1,170 @@
+whether our algorithm is supposed to use a quasi-probability
+distribution sampled from a quantum state (a list of
+quasi-probabilities), or an expectation value of a certain observable
+with respect to a quantum state (a real number).
+
+A probability distribution is often of interest in optimization problems
+that return a classical bit string, encoding a certain solution to a
+problem at hand. In these cases, we might be interested in finding a bit
+string that corresponds to a ket value with the largest probability of
+being measured from a quantum state, for example.
+
+An expectation value of an observable could be the target quantity in
+scenarios where the knowledge of a quantum state is not relevant. This
+often occurs in optimization problems or chemistry applications, where
+the extremal energy of a system is to be discovered, for example.
+
+.. raw:: html
+
+   </details>
+
+.. raw:: html
+
+  <details>
+  <summary>Which parts of my code do I need to refactor?</summary>
+
+Replace all dependencies on ``QuantumInstance`` and ``Backend`` with the
+implementation of the ``BaseEstimator``, ``BaseSampler``, or both
+primitives from the ``qiskit_ibm_runtime`` library.
+
+It is also possible to use local implementations, as shown in the
+`Amplitude estimation use case <migrate-e2e#amplitude>`__.
+
+Notably, for common scenarios it is not necessary to handle backends
+differently nor to construct expressions for expectation values
+manually.
+.. raw:: html
+
+   </details>
+
+.. _migrate-code:
+
+Migrate code
+--------------------------------------------
+
+We have identified key patterns of behavior and use cases with code examples to help you migrate code to Qiskit
+Runtime.
+
+.. note::
+
+   The key to writing an equivalent algorithm using Qiskit Runtime primitives is to remove all dependencies on ``QuantumInstance`` and ``Backend`` and replacing them with the implementation of the Estimator, Sampler, or both primitives from the ``qiskit_ibm_runtime`` library.
+
+It is also possible to use local implementations, as shown in the
+`Amplitude estimation use case <migrate-e2e#amplitude.html>`__.
+
+Notably, for common scenarios it is not necessary to handle backends
+differently nor to construct expressions for expectation values
+manually.
+
+The following topics are use cases with code migration examples:
+
+* `Use Estimator in an algorithm <migrate-estimator.html>`__
+* `Use Sampler in an algorithm <migrate-sampler.html>`__
+* `Use Estimator and Sampler in an algorithm <migrate-est-sam.html>`__
+* `Update parameter values while running <migrate-update-parm.html>`__
+* `Primitive-based routines <migrate-prim-based.html>`__
+* `End-to-end example <migrate-e2e.html>`__
+
+
+Related links
+-------------
+
+* `Use Estimator in an algorithm <../tutorials/how-to-getting-started-with-estimator>`__
+* `Use Sampler in an algorithm <../tutorials/how-to-getting-started-with-sampler>`__
```

Comment:
I think you meant to call them "Getting started with...", otherwise they'd have the same text as lines 158&159.